config/consensus.go

```diff
@@ -953,6 +961,16 @@ func initConsensusProtocols() {
 	vFuture.MaxExtraAppProgramPages = 3
 	vFuture.MaxAppProgramLen = 2048
 
+	// Individual limits raised
+	vFuture.MaxAppTxnForeignApps = 8
+	vFuture.MaxAppTxnForeignAssets = 8
+	// but MaxAppTxnReferences is unchanged.
+
+	// MaxAppTxnAccounts has not been raised yet.  It is already
+	// higher (4) and there is a multiplicative effect in
+	// "reachability" between accounts and creatables, so we
+	// retain 4 x 4 as worst case.
```

Comment:
```suggestion
	// retain 4 x 8 as worst case.
```
4 accounts and 8 creatables each?

Reply:
Since the total (accounts, asa, apps) is 8, the worst case is splitting that total in half, 4 for account, 4 for asa/app.

Reply:
ah, right, got confused